tests/test-team/template-mgmt-component-tests/template-mgmt-choose-page.component.ts

```diff
@@ -169,3 +169,180 @@ test.describe('Choose Template Type Page', () => {
+    await chooseTemplatePage.clickContinueButton();
+    await page.waitForTimeout(3000);
+    await expect(page.locator('.nhsuk-error-summary')).toBeVisible();
+    await expect(page.locator('.nhsuk-error-summary')).toHaveText([
+      'There is a problemEnter a template nameEnter a template message',
+    ]);
+  });
+
+  test('NHS App Message template populated and continued to the preview screen displayed', async ({
+    page,
+    baseURL,
+  }) => {
+    const chooseTemplatePage = new TemplateMgmtChoosePage(page);
+
+    await chooseTemplatePage.navigateToChooseTemplatePage(emptySessionData.id);
+
+    await expect(page).toHaveURL(
+      `${baseURL}/templates/choose-a-template-type/${emptySessionData.id}`
+    );
+    expect(await chooseTemplatePage.fieldsetHeading.textContent()).toBe(
+      'Choose a template type to create'
+    );
+    await TemplateMgmtChoosePage.checkRadioButton(
+      page.locator('[id="templateType-NHS_APP"]')
+    );
+    await chooseTemplatePage.clickContinueButton();
+    await expect(page.locator('h1')).toHaveText(
+      'Create NHS App message template'
+    );
+    const templateName = 'NHS Testing 123'; // Replace with the text you want to enter
+    await page.locator('[id="nhsAppTemplateName"]').fill(templateName);
+    const templateMessage = 'Test Message box'; // Replace with the text you want to enter
+    await page.locator('[id="nhsAppTemplateMessage"]').fill(templateMessage);
+    await chooseTemplatePage.clickContinueButton();
+    await expect(page.locator('h1')).toHaveText(
+      'NHS App message templateNHS Testing 123'
+    );
+  });
+
+  test('Hyperlinks & back button functionality', async ({ page, baseURL }) => {
+    const chooseTemplatePage = new TemplateMgmtChoosePage(page);
+
+    await chooseTemplatePage.navigateToChooseTemplatePage(emptySessionData.id);
+
+    await expect(page).toHaveURL(
+      `${baseURL}/templates/choose-a-template-type/${emptySessionData.id}`
+    );
+    expect(await chooseTemplatePage.fieldsetHeading.textContent()).toBe(
+      'Choose a template type to create'
+    );
+    await TemplateMgmtChoosePage.checkRadioButton(
+      page.locator('[id="templateType-NHS_APP"]')
+    );
+    await chooseTemplatePage.clickContinueButton();
+    const footerLinks = [
+      {
+        name: 'Home',
+        selector: 'a[data-testid="accessibility-statement-link"]',
+      },
+      { name: 'Contact Us', selector: 'a[data-testid="contact-us-link"]' },
+      { name: 'Cookies', selector: 'a[data-testid="cookies-link"]' },
+      {
+        name: 'Privacy Policy',
+        selector: 'a[data-testid="privacy-policy-link"]',
+      },
+      {
+        name: 'Terms and Conditions',
+        selector: 'a[data-testid="terms-and-conditions-link"]',
+      },
+    ];
+
+    await Promise.all(
+      footerLinks.map((link) =>
+        expect(
+          page.locator(link.selector),
+          `${link.name} should be visible`
+        ).toBeVisible()
+      )
+    );
```

Comment:
you are just checking whether stuff exists on the page - we already do this in the unit tests, i'm not sure that this adds anything?

Reply:
We would like to leave this in for now to test links later. thanks